pom.xml

```diff
@@ -20,6 +20,7 @@
         <javax-version>7.0</javax-version>
         <mockito-core-version>2.19.0</mockito-core-version>
         <junit-version>4.12</junit-version>
+        <jacoco-version>0.7.5.201505241946</jacoco-version>
```

Comment:
that's a lot of numbers.  I'm sure there is a version without all of the digits on the end

Reply:
I just went with the version it gave me lol I'll trim it

Reply:
version should not have all of the trailing numbers.  Let's get a version that doesn't have that